cmd/main.go

```diff
@@ -34,16 +36,29 @@ func main() {
 		logger.Sugar().Fatalf("cannot connect to auth service", err)
 	}
 
+	checkinConn, err := grpc.NewClient(conf.Svc.CheckIn, grpc.WithTransportCredentials(insecure.NewCredentials()))
+	if err != nil {
+		logger.Sugar().Fatalf("cannot connect to checkin service", err)
+	}
+
 	authClient := authProto.NewAuthServiceClient(authConn)
 	authSvc := auth.NewService(authClient, logger)
 	authHdr := auth.NewHandler(authSvc, validate, logger)
 
+	checkinClient := checkinProto.NewCheckInServiceClient(checkinConn)
+	checkinSvc := checkin.NewService(checkinClient, logger)
+	checkinHdr := checkin.NewHandler(checkinSvc, validate, logger)
+
 	r := router.New(conf, corsHandler, authMiddleware)
 
 	r.V1Get("/auth/google-url", authHdr.GetGoogleLoginUrl)
 	r.V1Post("/auth/verify-google", authHdr.VerifyGoogleLogin)
 	r.V1Post("/auth/test", authHdr.Test)
 
+	r.V1Post("/checkin", checkinHdr.Create)
+	r.V1Post("/checkin/userId/", checkinHdr.FindByUserID)
+	r.V1Post("/checkin/email/", checkinHdr.FindByEmail)
```

Comment:
let's do it this way; it's more conventional
/checkin/:userId
/chcekin/email/:email

Reply:
🫡